contracts/services/QueryService.sol

```diff
@@ -350,6 +350,42 @@ contract QueryService is IVersioned {
         return response; 
     }
 
+    function getAssetBalancesForIssuer(
+        address issuer,
+        address investor,
+        address[] memory assetFactories,
+        address[] memory campaignFactories
```

Comment:
I'm not sure why there is `campaignFactories` if you don't use them. Aren't just `assetFactories` sufficient enough to list the assets in this phase?

---

When we talked about the issue where we would have campaigns that would sometimes have duplicate items in the response, couldn't we just fill the map (`mapping(uint => Structs.AssetBalance)` where the key would be the token address so we wouldn't have duplicates? 
I posted it here just as a reminder.

Reply:
Add comment why `campaignFactories` is not used.